src/components/layout/Footer.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-len */
+import styledComponents from "styled-components"
+
+
+const Footer = () => {
+    return (
+        <FooterBox>
+            <FooterH3><i className='fa fa-tree'/> Wood Home</FooterH3>
+            <FooterDescription>Ecommerce created during Wizeline’s Academy React Bootcamp. Follow us!</FooterDescription>
+            <SocialNetwork>
+                <SocialNet><Icon className="fa fa-facebook" /></SocialNet>
+                <SocialNet><Icon className="fa fa-twitter" /></SocialNet>
+                <SocialNet><Icon className="fa fa-linkedin-square" /></SocialNet>
+            </SocialNetwork>
+        </FooterBox>
+
+    )
+}
+
+const FooterBox = styledComponents.div`
```

Comment:
To keep components cleaner, I would recommend to create a folder called "styles" and create all the styled components in that folder. The file of the styled components could be called something like:
SocialNetwork.styled.js and the just export that component and import it in this file.